Apsitvarkom.DataAccess/DbInitializer.cs

```diff
@@ -9,7 +9,15 @@ public static class DbInitializer
 {
     public static void InitializePollutedLocations(PollutedLocationContext context)
     {
-        var fakePollutedLocations = new PollutedLocation[]
+        var fakePollutedLocations = CreateFakePollutedLocationArray();
+
+        context.PollutedLocations.AddRange(fakePollutedLocations);
+        context.SaveChanges();
+    }
+
+    public static PollutedLocation[] CreateFakePollutedLocationArray()
```

Comment:
This is just a getter, right? No need for a whole method:

`public static PollutedLocation[] FakePollutedLocationArray => new PollutedLocation[] {...}`

(no need for `Create` in the name as well)

Reply:
changed it to be a lazy instead to fulfill the requirement, so sticked with the now private static method instead of it simply being an array